cohortextractor/backends/base.py

```diff
@@ -23,6 +23,7 @@ def __init_subclass__(cls, **kwargs):
         # Register each Backend by its id so we can identify it from an environment variable
         register_backend(cls)
         # Make sure each Backend knows what its tables are
+        cls.tables = set()
```

Comment:
If you really wanted to be belt and braces you could do:
```py
tables = set()
...
cls.tables = frozenset(tables)
```
But that's probably not necessary.